src/app/utils/externalLinks.ts

```diff
@@ -1,14 +1,37 @@
 import { Layer } from '../../oasis-nexus/api'
 
-export const socialMedia = {
-  telegram: 'https://t.me/oasisprotocolcommunity',
-  twitter: 'https://twitter.com/oasisprotocol',
-  discord: 'https://oasis.io/discord',
-  // This API link is for testing if invite is still valid.
-  isDiscordStillValid: 'https://oasis.io/discord/invite-api-check',
-  youtube: 'https://www.youtube.com/channel/UC35UFPcZ2F1wjPxhPrSsESQ',
-  reddit: 'https://www.reddit.com/r/oasisnetwork/',
-}
+const customTelegram = process.env.REACT_APP_SOCIAL_TELEGRAM
+const customTwitter = process.env.REACT_APP_SOCIAL_TWITTER
+const customDiscord = process.env.REACT_APP_SOCIAL_DISCORD
+const customYouTube = process.env.REACT_APP_SOCIAL_YOUTUBE
+const customReddit = process.env.REACT_APP_SOCIAL_REDDIT
+const customLinkedin = process.env.REACT_APP_SOCIAL_LINKEDIN
+const customDocs = process.env.REACT_APP_SOCIAL_DOCS
+const customHome = process.env.REACT_APP_SOCIAL_HOME
```

Comment:
Remove `custom` wording, as it should default to Oasis anyway.

Reply:
Yeah, removed those consts.